packages/subgraph/src/packages/token/token-voting.ts

```diff
@@ -76,9 +78,7 @@ export function _handleProposalCreated(
       const action = actions[index];
 
       let actionId =
-        event.address.toHexString() +
-        '_' +
-        event.params.proposalId.toHexString() +
+        generateProposalId(event.address, pluginProposalId) +
         '_' +
```

Comment:
Just commenting here, but let's start using `concat` from now on. In order not to make the PR huge, only use `concat` for things that you did in the PR anyways. in (`getProposal`, for `actionId`)

Reply:
amended